rir/src/compiler/analysis/verifier.cpp

```diff
@@ -214,6 +225,21 @@ class TheVerifier {
                 }
             });
             if (slow) {
+                std::unordered_set<BB*> inp;
+                for (auto in : cfg(bb->owner).immediatePredecessors(bb))
+                    inp.insert(in);
+                phi->eachArg([&](BB* bb, Value*) { inp.erase(inp.find(bb)); });
```

Comment:
this can backfire if bb is not found:
> The iterator pos must be valid and dereferenceable. Thus the end() iterator (which is valid, but is not dereferenceable) cannot be used as a value for pos.